src/app/component/MissionCard.jsx

```diff
@@ -76,13 +90,72 @@ const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest })
 
   return (
     <Card className={classes.root} {...rest}>
-      <CardHeader
-        title={title}
-        titleTypographyProps={{ variant: "h2", component: "span" }}
-        className={classes.cardHeader}
-      />
       <CardContent className={classes.cardContent}>
-        <MissionCardContent contentItems={contentItems} classes={classes} />
+        <Grid container spacing="1" alignItems="flex-start" justify="flex-end" direction="row">
+          <Grid item>
+            <img height="20" src={appleIcon} alt="" />
+          </Grid>
+          <Grid item style={{ flex: 1 }}>
+            {title}
```

Comment:
![image](https://user-images.githubusercontent.com/166867/81172551-8f76b300-8ff2-11ea-913f-89189c3fb37d.png)

Looking at the design it'd be great if we could get write a method that goes from FoodBoxDetails to something like "2 x Fruit & Veggies Medly" instead of just "No title supplied." here. 

From this data structure to a string. Tricky because we don't know what kind of details but we can put a method of "Mission" that generates a title, maybe? And then a switch on MissionType that then assumes if MissionType.FoodBox that we will have a FoodBoxDetails schema..

https://github.com/factn/resilience-app/blob/57030727a133e69ba4eca24520211c426268447c/src/app/model/schema.ts#L151

--

The complication is we may have more than one type of box. In practice, for now, it will always be the same one but to handle it in a general way maybe we can just do a "..." type thing.

So generate a title like "2 x Fruit & Vege Medly, 1 x Care Pack" but then cut it off so you get "2 x Fruit & Vege Med..." as the title? 

--

Also I think it'd be great to get the background to be grayed out or similar - we want people to click on the title  I think -  it will make it much more usable ?